checkers/rules/rules.go

```diff
@@ -160,10 +160,22 @@ func emptyStringTest(m dsl.Matcher) {
 	m.Match(`len($s) != 0`).
 		Where(m["s"].Type.Is(`string`)).
 		Report("replace `$$` with `$s != \"\"`")
+	m.Match(`len($s) >= 0`).
+		Where(m["s"].Type.Is(`string`)).
+		Report("replace `$$` with `$s != \"\"`")
+	m.Match(`len($s) > 0`).
+		Where(m["s"].Type.Is(`string`)).
+		Report("replace `$$` with `$s != \"\"`")
 
 	m.Match(`len($s) == 0`).
 		Where(m["s"].Type.Is(`string`)).
 		Report("replace `$$` with `$s == \"\"`")
+	m.Match(`len($s) <= 0`).
+		Where(m["s"].Type.Is(`string`)).
+		Report("replace `$$` with `$s == \"\"`")
+	m.Match(`len($s) < 0`).
```

Comment:
This is a `badCond` client, isn't it?
len should never return any negative values, so `len(x) < 0` should always be false.

Reply:
Yep, you're absolutely correct. Fixed.